pom.xml

```diff
@@ -140,6 +139,202 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>io.sundr</groupId>
+                <artifactId>sundr-maven-plugin</artifactId>
+                <version>0.200.0</version>
+                <executions>
+                    <execution>
+                        <inherited>false</inherited>
+                        <goals>
+                            <goal>generate-bom</goal>
+                        </goals>
+                        <configuration>
+                            <boms>
+                                <bom>
+                                    <artifactId>langchain4j-bom</artifactId>
+                                    <name>LangChain4j :: BOM</name>
+                                    <description>Bill of Materials POM for getting full, complete set of compatible versions of LangChain4j modules</description>
+                                    <properties>
+                                        <skipStagingRepositoryClose>true</skipStagingRepositoryClose>
+                                        <sonar.skip>true</sonar.skip>
+                                    </properties>
+                                    <extraDependencies>
```

Comment:
Why are you addding those as extra dependencies?
If they are available in the project, or transitives you could add them using something like: https://github.com/fabric8io/kubernetes-client/blob/main/pom.xml#L1236

Reply:
I don't think they are available in the project nor as transitives